learn/app-development/app-security/session-persistence.md

```diff
@@ -57,4 +57,18 @@ For JDBC, it displays a dropdown of all the imported databases in the project. T
 Ensure the selected database should be in editable mode and not read-only.
 :::
 
-2. **Save** the settings.
\ No newline at end of file
+2. **Save** the settings.
+
+
+### Mongo DB
+
+You can configure a MongoDB database for storing sessions. 
+
+1. From the Security window, choose **Session Persistence**
+2. Choose the **Type** `MONGODB` from the dropdown.
+3. Provide all the required details, such as Database name, Port, Host, Username, and Password. 
+4. Click **Save**.
+
+The required driver and files get imported into the project, and the application can connect to MongoDB.  
```

Comment:
Will the required drive files get auto added? @AmairK00 @swethakundaram If yes, clarify that